### DISCUSSION TITLE
fcitx5中如何添加双拼方案

### DISCUSSION THREAD

**Discussion by ghost** [fcitx/fcitx5#495] [Q&A] [answered]
你好，我在fcitx设置界面找到了双拼方案的选择，如小鹤、微软、等等，我想加入其它的双拼方案，是否可以？

**Comment by wengxt** [comment, marked as answer]
可以在 ~/.config/fcitx5/pinyin/sp.dat 写一个自定义的双拼方案，配置选自定义

例子可以参考
https://github.com/fcitx/libime/blob/20c8221a9c19b292bcc10ad7b2453b74ec512f7d/test/testshuangpinprofile.cpp#L111
https://github.com/fcitx/libime/blob/20c8221a9c19b292bcc10ad7b2453b74ec512f7d/test/testshuangpinprofile.cpp#L167

**Reply by ghost** [reply]
如果想重新插入一个方案是不是要重新编译？

**Reply by ghost** [reply]
我的意思是会显示方案名称。。。

**Reply by ghost** [reply]
试了一下不起作用，那个分号要保留吗“ ？？？

**Reply by ghost** [reply]
搞定了，文件名写错了。
